### PR TITLE
Fixed Issue #61 

### PR DESCRIPTION
**Pull request by SpeedSoop**
Someone plz approve

### CODE REVIEW

**Comment by ivanccheng** [comment]
hi, did you do any testing on the admin page?

**Comment by choiii-darren** [comment]
i fixed some merge conflicts
pull and test again

File: front-end/src/components/Results.js [hunk 1]
```diff
@@ -55,24 +54,26 @@ const ResultsPage = () => {
 
     // TODO: someone with React experience pls do this better
     function formatPlayerSingles(match, index) {
-        let player = players.find(x => x.id === match.players[index]).first_name
+        let player = players.find(x => x.id === match.players[index])
+        let playerString = player.first_name + " " + player.last_name 
 
+    console.log(playerString)
```
Comment (ivanccheng) [inline front-end/src/components/Results.js:60]: don't need this anymore

**Review by ivanccheng — COMMENTED** [review]
looks good, also deployed to take a look

nice fix on the admin page, before if you accidentally clicked a match it would get deleted, now only the button will do that

**Review by choiii-darren — APPROVED** [review]
remove any console.logs and we're good

**Pull request merged**